src/main/resources/jenkins/plugins/http_request/HttpRequestStep/help-contentType.html

```diff
@@ -0,0 +1,3 @@
+<div>
+	Add 'Content-Type: foo' HTTP request header where 'foo' is the HTTP content type the request is using.
```

Comment:
Note that `'` won't render well. please use single quote marks or Annotate values with `<code>`

Reply:
@oleg-nenashev  Thanks, I've made the changes.